packages/bsb-arbor/docs/index.rst

```diff
@@ -14,3 +21,6 @@ BSB Arbor documentation
    bsb_arbor/modules
    genindex
    py-modindex
+
+`Main BSB documentation <https://bsb.readthedocs.io/en/latest/index.html>`_
+---------------------------------------------------------------------------
```

Comment:
Didn't you add footer stuff to `sphinxext-bsb` for this?

Reply:
Yes I did but the logos are tiny and not very explicit about their purpose, so I prefer to have also this more explicit solution.

Reply:
ok, i see :)